frontend/packages/client/src/components/CommunityCreate/StepTwo.js

```diff
@@ -76,11 +103,28 @@ export default function StepTwo({
     onDataChange({ listAddrAuthors: [...listAddrAuthors, { addr: "" }] });
   };
 
+  const openModalHelp = () => {
+    openModal(<HelpModal />, {
+      classNameModalContent: "rounded-sm",
+    });
+  };
+
   return (
     <>
       <CommunityUsersForm
-        title="Admins"
-        description="The admins will be able to edit the space settings and moderate proposals. You must add one address per line."
+        title={
+          <>
+            Admins
+            <button
+              className="delete has-text-grey rounded-full cursor-pointer"
+              style={buttonStyle}
+              onClick={openModalHelp}
+            >
+              ?
+            </button>
+          </>
+        }
+        description="The admins will be able to edit the space settings and moderate proposals. You must add one address per line. You must add one address per line."
```

Comment:
`You must add one address per line.` is duplicated at the end of the line

Reply:
good catch! fixed